randovania/game_description/db/region.py

```diff
@@ -50,16 +46,16 @@ def area_by_name(self, area_name: str, is_dark_aether: bool | None = None) -> Ar
         raise KeyError(f"Unknown name: {area_name}")
 
     def area_by_identifier(self, location: AreaIdentifier) -> Area:
-        if self.name != location.world_name:
-            raise ValueError(f"Attempting to use AreaIdentifier for {location.world_name} with world {self.name}")
+        if self.name != location.region_name:
+            raise ValueError(f"Attempting to use AreaIdentifier for {location.region_name} with region {self.name}")
         return self.area_by_name(location.area_name)
 
-    def correct_name(self, in_dark_world: bool) -> str:
-        if in_dark_world and self.dark_name is not None:
+    def correct_name(self, use_dark_name: bool) -> str:
```

Comment:
while we're in here renaming things, what do you think of renaming `dark_name` etc to something else? cave story kinda abuses this system although maybe I should just make it not abuse it instead lol

Reply:
I'm slowly collecting my courage to refactor Prime 2 to have different ~~World~~Regions for Light/Dark. I'm afraid it's gonna do _shenanigans_ with pickup distribution because the weighting is based on how filled up each region is.

Could be renamed "alternative_name" I guess?

Reply:
if we're both thinking about refactoring our games to do that, then maybe we should just leave it be? but yeah probably `alternative_name` (and should update `Area`'s `is_in_dark_aether` too while we're at it lol) if we do change it